FhotoFithFriends/src/main/resources/templates/index.html

```diff
@@ -13,7 +13,9 @@
     <div class="container">
         <h1 id="title">Fhoto Fith Friends</h1>
         <div id="home-screen">
-        <p id="stuff">Behold, My Stuff!</p>
+        	<video id="video" width="640" height="480" autoplay></video>
+			<button id="snap" class="btn btn-primary">Snap Photo</button>
+			<canvas id="canvas" width="640" height="480"></canvas>
```

Comment:
Added HTML to support camera, I think the permission prompt bug may originate in the loading of this code compared to when the JS loads. Changing autoplay might help.